File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] arr) {
        int min = Arrays.stream(arr).min().getAsInt();
        if(arr.length < 2) {
            return new int[]{-1};
        }
        int[] answer = new int[arr.length-1];
        int count = 0;
        for(int i = 0; i < arr.length; i++){
            if(arr[i] != min){
                answer[count++] = arr[i];
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.js
function solution(arr) {
    var answer = [];
    var min = arr[0];
    for(let i = 0; i < arr.length; i++){
        if(arr[i] < min){
            min = arr[i];
        }
    }
    for(let i = 0; i < arr.length; i++){
        if(arr[i] != min){
            answer.push(arr[i]);
        }
    }
    return answer.length == 0 ? [-1] : answer;
}